pkg/api/types.go

```diff
@@ -199,6 +199,9 @@ type Config struct {
 	// the container for images that don't have a tar binary available, if the
 	// image contains ONBUILD commands that would be executed.
 	BlockOnBuild bool
+
+	// HasOnBuild will be set to true if the builder image contains ONBUILD instructions
+	HasOnBuild bool
```

Comment:
I think in the future we should stop bloating the Config and refactor the code ;-) But I'm OK wit this for now.